util/data_generators.py

```diff
@@ -25,11 +25,16 @@ def growth_map(
 
 
 def generate_entry(conn, date: datetime, plant_id: str) -> None:
-    image_uri = uuid.uuid4().hex
+    # TODO in demo, put the location and range of an actual forest here
```

Comment:
Perhaps a way to pass these in so we aren't modifying code specifically for the demo?

Reply:
added :)